content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,382 @@
+   dell’indirizzo telematico dove ricevere tutte le fatture elettroniche`, insert Odoo's
+   :guilabel:`Codice Destinatario` `K95IV18`, and confirm.
+
+.. seealso::
+   :doc:`../../receivables/customer_invoices/electronic_invoicing`
 
-Give Odoo permission to process files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+File processing authorization (Odoo)
+------------------------------------
 
 Since the files are transmitted through Odoo's server before being sent to SDICoop or received by
-your database, you need to authorize Odoo to process your files from your database.
+your database, you need to authorize Odoo to process your files from your database. To do so, go to
+:menuselection:`Accounting --> Configuration --> Settings --> Electronic Document Invoicing`.
 
-To do this, go to :menuselection:`Accounting --> Settings --> Electronic Invoicing` and click on
-*Register*.
+There are **three** modes available:
 
-.. image:: italy/register.png
-   :align: center
-   :alt: Register button to allow Odoo to process invoices.
+- :guilabel:`Demo` mode simulates an environment in which invoices are sent to the government. In
+  this mode, invoices need to be *manually* downloaded as XML files and uploaded to the **Agenzia
+  delle Entrate**'s website;
+- :guilabel:`Test (experimental)` mode sends invoices to a non-production (i.e., test) service.
+  Invoices are sent from Odoo to selected *fake* destinations of the **Pubblica Amministrazione**'s
+  test servers. Saving this change directs all companies on the database to use this
+  configuration;
+- :guilabel:`Official` is a production mode that sends your invoices directly to the **Agenzia delle
+  Entrate**.
 
-.. Note::
-   All your files are encrypted upon reception in such a way that only you are able to decrypt them.
+Once a mode selected, you need to accept the **terms and conditions** by ticking :guilabel:`Allow
+Odoo to process invoices`, and then :guilabel:`Save`. You can now record your transactions in Odoo
+Accounting.
 
-Enable FatturaPA on the Sales journal
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. warning::
+   Selecting either :guilabel:`Test (experimental)` or :guilabel:`Official` is **irreversible**.
+   Once in :guilabel:`Official` mode, it is not possible to select :guilabel:`Test (experimental)`
+   or :guilabel:`Demo`, and same for :guilabel:`Test (experimental)`. We recommend creating a
+   separate database for testing purposes only.
 
-Open your sales journal. Under the *Advanced Settings* tab, in the **Electronic invoicing** field,
-check *FatturaPA* and *Save*.
+.. note::
+   :guilabel:`Codice Destinatario` of fake destinations: `0803HR0` - `N8MIMM9` - `X9XX79Z`.
 
-.. image:: italy/journal.png
+.. image:: italy/italy-edi.png
    :align: center
-   :alt: FatturaPA feature in the sales journal's settings
+   :alt: Italy's electronic document invoicing options
+
+Taxes
+=====
+
+Specific configurations are required for the **reverse charge** type of taxes. In case of a
```

Comment:
I'm not sure what new information (directly related to the loca) this suggestion would bring. The actual PR states that specific configurations (found underneath it) are required for reverse charge taxes. Explaining how it is integrated or how it works within Odoo is too general, in my opinion

Reply:
I don't think so, because taxes need to be configured also for cases like San Marino, and as soon as I finish the PR I'm working on (end of this week probably), for Enasarco taxes, for Withholding taxes, for Pension fund contribution taxes... ( next PR 😄 ) Of course you can omit the part "so that it can be easily used and maintained", now that I read it again it just sounds weird here.

Anyway it's somehow important that the user knows that we're implementing most features for e-Invoices out of taxes configurations, so that if they're not configured as stated below, they won't tell the Tax Agency the right things.